virttest/storage.py

```diff
@@ -170,7 +169,9 @@ def get_image_filename(params, root_dir, basename=False):
                 return iscsi.get_image_filename(portal, target, lun,
                                                 user, password)
         if enable_gluster:
-            return gluster.get_image_filename(params, image_name, image_format)
+            if storage_type == 'glusterfs-direct':
```

Comment:
Seems not need this check.

Reply:
By now, get_image_filename returns the gluster uri, used for gluster direct access. So this check is needed to differentiate glusterfs(mount glusterfs) and gluster-direct (uri access)